test/fastboot-config-test.js

```diff
@@ -0,0 +1,42 @@
+'use strict';
+
+const expect = require('chai').use(require('chai-string')).expect;
+const RSVP = require('rsvp');
+const request = RSVP.denodeify(require('request'));
+
+const AddonTestApp = require('ember-cli-addon-tests').AddonTestApp;
+
+describe('FastBoot config', function() {
+  this.timeout(400000);
```

Comment:
Any reason for such a high timeout?

Reply:
Just copy&paste from another test, I'll remove.

Reply:
I had to put back the timeout due to failing tests, not sure if the number is too high.